src/Carousel.css

```diff
@@ -3,5 +3,33 @@ ul {
   list-style-type: none;
   padding: 0;
   margin: 0;
+  justify-content: center;
 }
 
+
+.carousel {
+  max-width:60%;
+  padding:50px 50px 50px 50px;
+  position:relative;
+  overflow: hidden;
+}
+
+.button {
+  background: none;
+  border: none;
+}
+
+.fa-angle-left, .fa-angle-right {
```

Comment:
Just wondering if moving this to another file would solve the issue of the arrows moving with the cards. If we put it on app for example would they remain static?